gym_snake/envs/snake_env.py

```diff
@@ -12,22 +12,28 @@ class SnakeEnv(gym.Env):
     metadata = {'render.modes': ['human']}
 
     def __init__(self, 
-        use_pygame: bool = True, 
+        board_height: int = 10, 
+		board_width:int = 10, 
+		use_pygame: bool = True, 
+		fps: int = 25,
```

Comment:
I think default should be 3000 since faster is better right?

```suggestion
		fps: int = 3000,
```